lib/flex.js

```diff
@@ -0,0 +1,132 @@
+const Configuration = require('./configuration/configuration')
+const extractValidationStats = require('./stats/extractValidationStats')
+
+const SUPPORTED_VALIDATOR = [
+  'title',
+  'label',
+  'description',
+  'assignee',
+  'dependent',
+  'milestone',
+  'approval'
+]
+
+const SUPPORTED_ACTIONS = [
+  'comments',
+  'checks'
+]
+
+// Main logic Processor of mergeable
+const executeMergeable = async (context, registry) => {
```

Comment:
Where is this being called? And how? Is `/index.js` missing from this PR? Is `lib/mergeable.js` missing from this PR?